### PR TITLE
Add support for EezTire TPMS10ATC E618

### PR DESCRIPTION
**Pull request by ProfBoc75**
as discussed into the issue #2384

### CODE REVIEW

File: src/devices/tpms_eezrv.c [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    char id_str[7];
+    sprintf(id_str, "%02x%02x%02x", b[0], b[1], b[2]);
+    float pressure_kPa  = b[3]*2.5;
+    int temperature_C   = b[4]-50;
+    int flags = b[5];
+    char flags_str[3];
+    sprintf(flags_str, "%x", flags);
+    /* clang-format off */
+    data_t *data = data_make(
+        "model",                       "",    DATA_STRING, "EezTire TPMS10ATC",
```
Comment (zuckschwerdt) [inline src/devices/tpms_eezrv.c:85]: I guess "10ATC" is 10 Anti Theft + Console package.
The sensors is "618E", model should be `"EEZTire-618E"`

Also add like in other TPMS:
`            "type",             "",             DATA_STRING, "TPMS",`

File: src/devices/tpms_eezrv.c [hunk 2]
```diff
@@ -0,0 +1,116 @@
+/** @file
+    EezTire TPMS.
+
+    Copyright (C) 2023 ProfBoc75 and Gliebig
```
Comment (merbanan) [inline src/devices/tpms_eezrv.c:4]: Usually this is peoples real names but feel free to leave it as is.

File: src/devices/tpms_eezrv.c [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+Data layout:
+
+    PRE CC IIIIII PP TT FF 00
+
+- PRE : FFFF
+- C : 8 bit CheckSum
+- I: 24 bit ID
+- P: 8 bit pressue
+- T: 8 bit temperature
```
Comment (merbanan) [inline src/devices/tpms_eezrv.c:27]: Add units and scaling for temperature and pressure.

**Comment by merbanan** [comment]
Just a few nits. Do we have test samples? If so maintainer_update changes can be added to the PR and then we can merge.

**Comment by ProfBoc75** [comment]
> Just a few nits. Do we have test samples? If so maintainer_update changes can be added to the PR and then we can merge.

Yes, we have sample made by @Gliebig here : https://github.com/Gliebig/rtl_433_tests/tree/master/tests/EEZ%20Tire%20TPMS/Model%20618E

Corrections done,

**Comment by Gliebig** [comment]
Are you waiting for me to do something on my end? If so, I don't know what it is you need me to do.

**Comment by zuckschwerdt** [comment]
All good, I'll merge this soon.

**Pull request merged**